public/components/main-nav/main-nav.stache

```diff
@@ -11,27 +11,27 @@
 				<span class="icon-bar"></span>
 				<span class="icon-bar"></span>
 			</button>
-			<a href="{{routeUrl}}" class="navbar-brand">
+			<a href="{{ routeUrl }}" class="navbar-brand">
 				<img src="/img/bitcentive-logo-brand.png" alt="" class="logo">
 			</a>
 		</div>
 
 		<div id="navbar" class="navbar-collapse collapse in" aria-expanded="true">
 			<ul class="nav navbar-nav">
-				{{#if session}}
-					<li {{#routeCurrent page="dashboard"}}class="active"{{/routeCurrent}}><a href="{{routeUrl page='dashboard'}}" class="">Dashboard</a></li>
-					{{#if isAdmin()}}
-						<li {{#routeCurrent page="contributors"}}class="active"{{/routeCurrent}}><a href="{{routeUrl page="contributors"}}">Contributors</a></li>
-						<li {{#routeCurrent page="users"}}class="active"{{/routeCurrent}}><a href="{{routeUrl page="users"}}">Users</a></li>
-					{{/if}}
-				{{else}}
-					<li {{#routeCurrent page="home"}}class="active"{{/routeCurrent}}><a href="{{routeUrl}}">Home Page</a></li>
-				{{/if}}
+				{{# if(session) }}
```

Comment:
While _helper expressions_ like `{{if session}}` are still supported, we encourage everyone to use _call expressions_ like `{{# if(session) }}` instead.

Also, this changes the spacing to something I consider more readable.